Objects/tupleobject.c

```diff
@@ -705,6 +705,26 @@ tuple_new_impl(PyTypeObject *type, PyObject *iterable)
         return PySequence_Tuple(iterable);
 }
 
+static PyObject *
+tuple_vectorcall(PyObject *type, PyObject * const*args,
+                 size_t nargsf, PyObject *kwnames)
+{
+    if (kwnames && PyTuple_GET_SIZE(kwnames) != 0) {
+        PyErr_Format(PyExc_TypeError, "tuple() takes no keyword arguments");
```

Comment:
It would be nice to add a function similar to _PyArg_NoKeywords() but accept a kwnames tuple. But it should be done in a separated PR.